CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ VS Code v1.56
 
 - docs(install): add raspberry pi section #3376 @jsjoeio
 - docs(maintaining): add pull requests section #3378 @jsjoeio
+- docs(maintaining): add merge strategies section #3379 @jsjoeio
```

Comment:
while you're here, can you also fix my changelog entry below? :pray:  I forgot the PR number and author info :flushed: